Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -388,6 +414,13 @@ private bool IsFileParameterSet()
             return String.Equals(this.ParameterSetName, "File", StringComparison.OrdinalIgnoreCase);
         }
 
+        private bool OverrideSwitchParam(bool paramValue, string paramName)
+        {
+            return MyInvocation.BoundParameters.ContainsKey(paramName)
+                ? ((SwitchParameter)MyInvocation.BoundParameters[paramName]).ToBool()
```

Comment:
The idea is that if the user doesn't specify a switch, then use the value from the settings file. But if the user specifies the switch on the command line then use the command line argument. 

For example, if `IncludeDefaultRules` is set to `$true` in the settings file and if the user specifies `-IncludeDefaultRules:$false` on the command line, then the final value for `IncludeDefaultRules` will be set to `$false`